Formula/tt.rb

```diff
@@ -20,17 +20,24 @@ class Tt < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl@3"
   uses_from_macos "curl"
+  uses_from_macos "git"
   uses_from_macos "unzip"
   uses_from_macos "zip"
 
   def install
     ENV["TT_CLI_BUILD_SSL"] = "shared"
     system "mage", "build"
     bin.install "tt"
+    etc.install "tt.yaml.default"
```

Comment:
Why not just
```suggestion
    (etc/"tarantool").install "tt.yaml.default" => "tt.yaml"
```
?

`brew` makes sure not to clobber existing configuration files in `etc` so this is always safe to do. It will get installed as `tt.yaml.default` if a `tt.yaml` already exists.

Reply:
thx, changed